pkgs/by-name/dm/dm-mono/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation {
```
`rec` seems to be unused